src/hubcast/clients/github/client.py

```diff
@@ -105,3 +133,60 @@ async def get_prs(self, branch=None):
                 url = f"{url}?head={self.repo_owner}:{branch}"
                 prs_res = await gh.getitem(url)
                 return [pr["number"] for pr in prs_res]
+
+    async def post_comment(self, issue_number: int, body: str):
+        payload = {"body": body}
+
+        gh_token = await self.auth.authenticate_installation(
+            self.repo_owner, self.repo_name
+        )
+
+        async with aiohttp.ClientSession() as session:
+            gh = gh_aiohttp.GitHubAPI(session, self.requester, oauth_token=gh_token)
+
+            url = f"/repos/{self.repo_owner}/{self.repo_name}/issues/{issue_number}/comments"
+            await gh.post(url, data=payload)
+
+    async def react_to_comment(self, comment_id: int, reaction: str):
+        """Add an emoji reaction to a GitHub PR comment.
+        See `valid_reactions` for a list of emoji options.
+        """
+
+        valid_reactions = [
+            "+1",
+            "-1",
+            "laugh",
+            "hooray",
+            "confused",
+            "heart",
+            "rocket",
+            "eyes",
+        ]
```

Comment:
Maybe we should export this as a global variable so that you can query the valid reactions / see them in your editor when building on top of the client library.